benchflow-faban-client/src/main/java/cloud/benchflow/faban/client/exceptions/FabanClientThrowable.java

```diff
@@ -0,0 +1,18 @@
+package cloud.benchflow.faban.client.exceptions;
+
+/**
+ * @author vincenzoferme.
+ */
+public class FabanClientThrowable extends Throwable {
```

Comment:
Right, fixed in https://github.com/benchflow/benchflow/pull/442/commits/6658c1b28264aa730db3b8804c2187df8254f0c0